<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>License</class>
 <widget class="QDialog" name="License">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#License{background-color:#4E4E4E;border:1px solid black}</string>
  </property>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#303030;color:#0ff;border:1px solid black;font-size:24px</string>
   </property>
   <property name="text">
    <string> 许可</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>845</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#303030;border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>470</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>121</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{background-color:#4e4e4e;color:white;border:none;font-size:14px}
QListWidget::item:selected{backgroundcolor:#4e4e4e;color:#0ff;border:none;font-size:14px}</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>89</y>
     <width>481</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#4e4e4e;color:white;border:none;font-size:14px
</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>20</width>
     <height>291</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
